### ISSUE TITLE
Output SMPL PKL

### ISSUE THREAD

**Issue by nshreyasvi** [JunukCha/MultiPerson#10]
Hello,
Thank you for the amazing work. I was trying to extract the output SMPL `.pkl` file in the standard SMPL format with `body_pose`, `global_orient` etc. as the keys. However, I see that the output contains `thetas` which is `(72.)` shape. I wanted to know if this is the same as `global_pose` ? I would also like to know how could I convert them to be in the `(1, 24, 3)` or `(1, 72)` format? I would also like to know if these axis angles are the `global_pose` or the `body_pose` (Camera Pose)?
I also see if I save the smpl output I get a key called ['joints_smpl24'] which seems to be of shape `[3, 24, 3]`. I would like to know fi this is the same as global `body_pose`?

**Comment by JunukCha** [comment]
I don't know exact what global pose means. But `thetas` which is 72 dimension denotes the rotation angle for each joint (24). If you want to change the shape,  you can use the code `thetas.reshape(1, 24, 3)` as torch or numpy provided. `joints_smpl24` might means 24 joints which have SMPL topology.

Thanks.

**Comment by nshreyasvi** [comment]
Hello, thank you. I believe that for the evaluation, I am trying to convert the output that I get from MultiPerson into a `(1,24,3)` 
output format. However, the output that I see in `joints_smpl24` is in `(3, 24, 3)` which is a bit confusing as I am not sure why it is not `(1, 24, 3)` as I provided an input of just one frame to it.

**Update: I just manually extracted the `thetas` and `betas` from the model (which were in `(,72)` format and used the following code to generate the SMPL model with `(1,24,3)`.**

```
betas = torch.tensor(data['betas'], dtype=torch.float32).unsqueeze(0)
pred_pose = torch.tensor(data['thetas'], dtype=torch.float32).unsqueeze(0)

# Load SMPL model and get predicted joints
smpl_model = smplx.create(model_path=smpl_model_path, model_type='smpl', gender='male')
output = smpl_model(betas=betas, body_pose=pred_pose[:, 3:], global_orient=pred_pose[:, :3], return_verts=True)
pred_joints = output.joints.detach().cpu().numpy()[:, :num_joints, :]  # Extract 24 joints
```

I would like to know if the output Mesh and pose here is accurate to the one that I obtain from the MultiPerson output?

**Comment by JunukCha** [comment]
Hello, I think the maximum estimation number of human is set as **3**. Thus, the output shape is (**3**, 24, 3).

```
output = smpl_model(betas=betas, body_pose=pred_pose[:1, 3:], global_orient=pred_pose[:1, :3], return_verts=True)
```

`pred_pose[:1]` means that you will use only one batch (one person) of `pred_pose`.
The above code might works for you. It would output of one person.

Thanks.

**Comment by nshreyasvi** [comment]
Hello,
Thank you. I used the pred_pose to simplify
